tasks/build-examples.js

```diff
@@ -82,7 +82,9 @@ function augmentExamples(files, metalsmith, done) {
       if (!(jsFilename in files)) {
         throw new Error('No .js file found for ' + filename);
       }
-      var jsSource = files[jsFilename].contents.toString();
+      var jsSource = files[jsFilename].contents.toString()
+          // Change data paths to absolute urls
+          .replace(/'data\//g, '\'http://openlayers.org/en/v' + pkg.version + '/examples/data/');
```

Comment:
Should we maybe do the same for `/resources`?
https://github.com/openlayers/ol3/tree/master/examples/resources